lib/Carton.pm

```diff
@@ -116,6 +121,14 @@ C<--deployment> option, you can avoid querying for a database like
 CPAN Meta DB or downloading files from CPAN mirrors upon deployment
 time.
 
+As an alternative to installing Carton on remote machine(s), you can
+use L<carton fatpack|Carton::Doc::Fatpack> to pack Carton into
```

Comment:
the `fatpack` command is known to be fragile as seen in https://github.com/perl-carton/carton/commit/9122643dfbbb5a919e505810eeaaae938ab8b8d0 and i'd like to recommend against using it.